src/hooks/useOttAnalytics.ts

```diff
@@ -9,44 +9,45 @@ const useOttAnalytics = (item?: PlaylistItem, feedId: string = '') => {
   const user = useAccountStore((state) => state.user);
 
   // ott app user id (oaid)
-  const oaid: number | undefined = user?.id ? Number(user?.id) : undefined;
+  const oaid: number | undefined = user?.id ? Number(user.id) : undefined;
 
   const [player, setPlayer] = useState<jwplayer.JWPlayer | null>(null);
 
-  const timeHandler = useCallback(({ position, duration }: jwplayer.TimeParam) => {
-    window.jwpltx.time(position, duration);
-  }, []);
+  useEffect(() => {
+    if (!window.jwpltx || !analyticsToken || !player || !item) {
+      return;
+    }
 
-  const seekHandler = useCallback(({ offset, duration }) => {
```

Comment:
Would be great if we could use a different target branch to avoid changes from another PR.

Reply:
These changes are from another [PR ](https://github.com/jwplayer/ott-web-app/pull/312)
Since the changes are pretty much in the same place that's why I've created the branch from there.